.circleci/config.yml

```
@@ -38,67 +51,127 @@ commands:
             choco install dotnet-sdk --version <<parameters.version>> -y --no-progress
           name: Install .NET Core Latest
-  
+
 jobs:
   build:
-    executor:
-      name: win/default
-      size: medium
-    # macos:
-    #   xcode: "11.3.0"
-    # Let's keep this disabled until we figure out the macOS build.
+    parameters:
+      os:
+        type: executor
+    executor: << parameters.os >>
     steps:
-      - run: git config --global core.symlinks true # https://github.com/git-for-windows/git/wiki/Symbolic-Links 
+      - when:
+          condition: 
+            equal: [ *windows, << parameters.os >>]
+          steps:
+            - run:
+                command: |
+                  git config --global core.symlinks true # https://github.com/git-for-windows/git/wiki/Symbolic-Links 
+                  nvm install lts
+                  rm "/c/Program Files/nodejs" -rf
+                  nvm use 16.16.0
       - checkout-update-submodules:
           paths: "NineChronicles.Headless"
       - yarn-install
-      - install-dotnet
-      - run: 
-          command: |
-            [Environment]::SetEnvironmentVariable("APV", (./scripts/make-apv.ps1), "User")
-          name: Generate APV
-      - run: 
-          command: |
-            set -ex
-            mkdir -p dist
-            curl -o "dist/config.json" 'https://download.nine-chronicles.com/9c-launcher-config.json'
-          shell: bash.exe
-          name: Copy config.json from the last release
-      - run: yarn sign-apv
+      - when:
+          condition:
+            equal: [ *macos, << parameters.os >>]
+          steps:
+            - run: 
+                command: |
+                  brew install --cask dotnet-sdk
+                  curl "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"
+                  sudo installer -pkg AWSCLIV2.pkg -target /
+                name: Install .NET Core, AWS
+      - when:
+          condition:
+            equal: [ *linux, << parameters.os >>]
+          steps:
+            - run: 
+                command: |
+                  wget https://packages.microsoft.com/config/ubuntu/22.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+                  sudo dpkg -i packages-microsoft-prod.deb
+                  rm packages-microsoft-prod.deb
+                  sudo apt update
+                  sudo apt install -y dotnet-sdk-6.0
+                name: Install .NET Core
       - run:
           command: |
             dotnet clean -c Release
             dotnet nuget locals all --clear
           working_directory: NineChronicles.Headless
           name: Clean Headless
       - run:
-          command: yarn concurrently npm:build-headless npm:bundle-player
-          shell: bash
+          command: yarn concurrently yarn:build-headless yarn:bundle-player
           name: Setup Software Components
-      - run:
+      - run: 
+          command: 'echo "::set-output name=apv::$(scripts/make-apv.sh)"'
```

```suggestion
      - run: 'echo "::set-output name=apv::$(scripts/make-apv.sh)"'
```

You can write like this if the step only has a command. You may want to name this though.

---

Added Name on it.